src/components/Votes.jsx

```diff
@@ -0,0 +1,44 @@
+import { useParams } from "react-router-dom";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons";
+import { updateArticleVotes } from "../assets/api-calls";
+
+function Votes({ initialVotes, count }) {
+  const { article_id } = useParams();
+  const [votes, setVotes] = useState(initialVotes);
+
+  const handleClickUp = () => {
+    updateArticleVotes(article_id, 1)
+      .then((updatedVotes) => {
+        setVotes(updatedVotes);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  const handleClickDown = () => {
+    updateArticleVotes(article_id, -1)
+      .then((updatedVotes) => {
+        setVotes(updatedVotes);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
```

Comment:
These 2 functions are extremely similar so could be refactored to be a single but more dynamic function.